Sources/SetNode.swift

```diff
@@ -7,33 +7,54 @@
 import Stencil
 
 class SetNode: NodeType {
+  enum Content {
+    case nodes([NodeType])
+    case value(Resolvable)
+  }
```

Comment:
Damn this quite confused me… `nodes` is actually used when rendering to a single string value, and `value` is used when pointing to a node tree/object… 😅

I now get the logic but I needed to read the code twice to understand and be sure you didn't invert the logic but my reading did ^^

Maybe renaming `case value` to `case reference(to:)` would have helped? Idk